### DISCUSSION TITLE
Integrating Open Media Vault 6?

### DISCUSSION THREAD

**Discussion by orionbg** [1000001101000/Debian_on_Intel_Terastations#11] [Q&A] [answered]
Hello,
Is it possible to integrate Open Media Vault 6 into this?
I've tried installing it after setup but am having a lot of errors during install and it does not work after that...
Thanks!

**Comment by 1000001101000** [comment]
If you use my Debian installer to install Debian on a Terastation you're left with a plain Debian install with a couple of patched drivers for gpio and fan sensors along with a few scripts to control the LCD and enable the hotswap bays by default by settings some gpios.

I'd expect any difficulties installing OMV afterward to be specific to their install stuff, possibly with solution(s) available in their docs/etc. 

If you wanted you could try installing with an OMV image and then retrofit my pieces into the resulting system instead as an alternative approach.

**Comment by orionbg** [comment]
Hey,
Thanks for the fast response.
I'm currently trying to get OMV6 to install properly on the NAS. Will try to figure it out.
BTW I'm using TS5400r and the image installed and booted without issues. just FYI

**Reply by 1000001101000** [reply, marked as answer]
That's good to hear!

One of the things I never thought about when setting up this project is any mechanism to learn about successful installs. Occasionally people ask me how many installs of these devices are out there or whether x.y.z device has ever had an install. The truth is I have no idea beyond a rough idea how many have reported issues.

**Reply by orionbg** [reply]
One more question if I may.
How can I control the fans on the NAS?
Currently, they are on full blast all the time and for a rack-mounted device those are very loud.
Thanks!

**Reply by orionbg** [reply]
an update on the Install...
I have tried several times to install OMV6 but there are always errors with some of the services crashing.
I'll try to install the OMV6 Image and then I'll have to install your addons.
Is there any script to install those automatically or some guide that a Windows person can follow, please?

**Reply by 1000001101000** [reply]
checkout the `fancontrol` package.

**Reply by 1000001101000** [reply]
All the components are in the repo and the scripts used to install them inside the installer are there too. I don't have anything better than that as far as install instructions.
